frontend/packages/ceph-storage-plugin/integration-tests/views/add-capacity.view.ts

```diff
@@ -1,33 +1,53 @@
-import { browser, ExpectedConditions as until, $ } from 'protractor';
+import { $, ExpectedConditions as until, browser } from 'protractor';
 import * as crudView from '@console/internal-integration-tests/views/crud.view';
 import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
 import { click } from '@console/shared/src/test-utils/utils';
 import { CAPACITY_UNIT, CAPACITY_VALUE, OCS_OP } from '../utils/consts';
-import { currentSelectors } from './installFlow.view';
+import { currentSelectors, VERSION } from './installFlow.view';
 
-export const ocsOp = $(`a[data-test-operator-row='${OCS_OP}']`);
+/**
+ * All generic selectors go into Defaults
+ * All OCPX.Y selectors that are not compatible with > X.(Y + 1) OCP goes into its own object.
+ * Everything else in DEFAULTS
+ */
 
-export const actionForLabel = (label: string) => $(`button[data-test-action='${label}']`);
-export const confirmButton = $('#confirm-action');
-export const storageClusterRow = (uid) => $(`tr[data-id='${uid}']`);
+const DEFAULTS = {
+  ocsOp: $(`a[data-test-operator-row='${OCS_OP}']`),
+  actionForLabel: (label: string) => $(`button[data-test-action='${label}']`),
+  confirmButton: $('#confirm-action'),
+  storageClusterRow: (uid: string) => $(`tr[data-id='${uid}']`),
+  getSCOption: (scName: string) => $(`a[id='${scName}-link']`),
+  capacityValueInput: $('input.ceph-add-capacity__input'),
+  totalRequestedcapacity: $('div.ceph-add-capacity__input--info-text strong'),
+  scDropdown: $('button[id="ceph-sc-dropdown"]'),
+  storageClusterNav: $('a[data-test-id="horizontal-link-Storage Cluster"]'),
+  getProgressingStateEl: (statusCol) => statusCol.$('span.co-icon-and-text'),
+  getReadyStateEl: (statusCol) => statusCol.$('span.co-icon-and-text span.co-icon-and-text span'),
+};
 
-const capacityValueInput = $('input.add-capacity-modal__input--width');
-const capacityUnitButton = $('button[data-test-id="dropdown-button"] .pf-c-dropdown__toggle-text');
-export const requestSizeValueInput = $('input[name="requestSizeValue"]');
+// TODO: NEHA, add support for other versions
+export const currentACSelector = (() => {
```

Comment:
```suggestion
export const currentSelector = (() => {
```

Reply:
We can't use the same selector name as its being defined inside install flow and few selectors of install flow have been used in add capacity flow.

Reply:
import {x as X} can be implied. No need to do in the same PR.

Reply:
yeah, correct.